src/lib/public/board/data/officers.json

```diff
@@ -1,718 +1,368 @@
+    }
   },
   {
     "fullName": "Eduardo Gomez",
-    "positions": {
-      "S21": {
-        "title": "nodebuds Officer",
-        "tier": 23
-      }
-    },
-    "picture": "eduardo-gomez.webp"
+    "picture": "eduardo-gomez.webp",
+    "positions": { "S21": { "title": "nodebuds Officer", "tier": 22 } }
   },
   {
     "fullName": "Nurhaliza Hassan",
+    "picture": "nurhaliza-hassan.webp",
     "positions": {
-      "S22": {
-        "title": "Marketing Director",
-        "tier": 12
-      },
-      "F22": {
-        "title": "Marketing Co-Lead",
-        "tier": 23
-      }
-    },
-    "picture": "nurhaliza-hassan.webp"
+      "S22": { "title": "Marketing Director", "tier": 8 },
+      "F22": { "title": "Marketing Co-Lead", "tier": 8 }
+    }
   },
   {
     "fullName": "Lisa Hong",
-    "positions": {
-      "S21": {
-        "title": "Create Officer",
-        "tier": 17
-      }
-    },
-    "picture": "lisa-hong.webp"
+    "picture": "lisa-hong.webp",
+    "positions": { "S21": { "title": "Create Officer", "tier": 16 } }
   },
   {
     "fullName": "Joshua Hughes",
-    "positions": {
-      "S21": {
-        "title": "ICC Representative",
-        "tier": 8
-      }
-    },
-    "picture": "joshua-hughes.webp"
+    "picture": "joshua-hughes.webp",
+    "positions": { "S21": { "title": "ICC Representative", "tier": 23 } }
   },
   {
     "fullName": "Iftekharul Islam",
-    "positions": {
-      "S22": {
-        "title": "Algo Officer",
-        "tier": 15
-      }
-    },
-    "picture": "iftekharul-islam.webp"
+    "picture": "iftekharul-islam.webp",
+    "positions": { "S22": { "title": "Algo Officer", "tier": 12 } }
   },
   {
     "fullName": "Ibrahim Israr",
-    "positions": {
-      "S22": {
-        "title": "Secretary",
-        "tier": 3
-      }
-    },
-    "picture": "ibrahim-israr.webp"
+    "picture": "ibrahim-israr.webp",
+    "positions": { "S22": { "title": "Secretary", "tier": 3 } }
   },
   {
     "fullName": "Joel Anil John",
+    "picture": "joel-anil-john.webp",
     "positions": {
-      "S22": {
-        "title": "Dev Officer",
-        "tier": 21
-      },
-      "F22": {
-        "title": "Algo Officer",
-        "tier": 15
-      }
-    },
-    "picture": "joel-anil-john.webp"
+      "S22": { "title": "Dev Officer", "tier": 19 },
+      "F22": { "title": "Algo Officer", "tier": 12 }
+    }
   },
   {
     "fullName": "Andy Lasso",
-    "positions": {
-      "F21": {
-        "title": "Dev Officer",
-        "tier": 21
-      }
-    },
-    "picture": "andy-lasso.webp"
+    "picture": "andy-lasso.webp",
+    "positions": { "F21": { "title": "Dev Officer", "tier": 19 } }
   },
   {
     "fullName": "Andrew Lau",
-    "positions": {
-      "S21": {
-        "title": "Treasurer",
-        "tier": 4
-      }
-    },
-    "picture": "andrew-lau.webp"
+    "picture": "andrew-lau.webp",
+    "positions": { "S21": { "title": "Treasurer", "tier": 4 } }
```

Comment:
The updated "tier" for Treasurer is '3'

` "Treasurer": { "id": 3, "index": 250 },`